src/sockets/commands/helperFunctions.ts

```diff
@@ -0,0 +1,22 @@
+import { Command } from './types';
+
+export function createCommandWithNewPrefix(
+  command: Command,
+  oldPrefix: string,
+  newPrefix: string,
+): Command {
```

Comment:
To be honest, I'd probably hard code 'm' -> 'p' in this function and call it convertModCommandToPercivalCommand. This way others can't misuse this function.